dev/integration_tests/flutter_gallery/pubspec.yaml

```diff
@@ -37,7 +37,7 @@ dependencies:
   typed_data: 1.3.0-nullsafety.3 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   url_launcher_linux: 0.0.1+1 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
   url_launcher_macos: 0.0.1+8 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
-  url_launcher_platform_interface: 1.0.8 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
+  url_launcher_platform_interface: 1.0.9 # THIS LINE IS AUTOGENERATED - TO UPDATE USE "flutter update-packages --force-upgrade"
```

Comment:
Doesn't this version need to be pinned?

Reply:
I do not know, @mdebbar or @ditman may know what's going on there. I'm assuming by the time the tree is open again that will be resolved.

Reply:
[This PR](https://github.com/flutter/plugins/pull/3155) will fix this issue. I'm working now on getting it green so I can land it.

Reply:
Closing the loop here: Mouad [published](https://pub.dev/packages/url_launcher_web/changelog#015) the fix for the build issue. Apologies for the breakage!